main.php

```diff
@@ -71,6 +70,21 @@
 		exit(1);
 	}
 }
+// Allow a first class env var to be set that disables .env file loading
+if (!getenv('SS_IGNORE_DOT_ENV')) {
```

Comment:
We can't have it the other way because this is designed for when setting env vars is tricky, so requiring an env var to be set to load the polyfill would be self-defeating.